tf_seal/cc/ops/seal_ops.cc

```diff
@@ -72,3 +72,18 @@ REGISTER_OP("SealPolyEval")
     .Input("pub_key: variant")
     .Output("out: variant")
     .SetIsStateful();
+
+REGISTER_OP("SealSavePublickey")
+    .Input("pub_key: variant")
+    .Output("out:variant")
+    .SetIsStateful();
+
+REGISTER_OP("SaveSecretkey")
+    .Input("secretkey: variant")
+    .Output("out:variant")
+    .SetIsStateful();
+
+REGISTER_OP("SaveCipherText")
```

Comment:
same: `SealSaveCiphertext`?

Reply:
After Public and Secret I want to save the Ciphertext

Reply:
Done